themes/bootstrap3/templates/search/facet-list.phtml

```diff
@@ -28,36 +26,45 @@
+          <input class="ajax_params" type="hidden" data-params="<?=$this->htmlSafeJsonEncode($ajaxParams)?>">
+        </form>
+      </div>
+    <?php endif; ?>
   </div>
 <?php endif; ?>
-<div class="lightbox-scroll full-facets">
+
+<?php $this->headScript()->appendFile('facets.js'); ?>
+
+<div id="facet-info-result" class="lightbox-scroll full-facets">
   <?php foreach ($this->sortOptions as $key => $sort): ?>
     <?php $active = $this->sort == $key; ?>
-    <div class="full-facet-list facet-group<?php if (!$active): ?> hidden<?php endif; ?>" id="facet-list-<?=$this->escapeHtmlAttr($key) ?>">
-      <?php if ($active): ?>
-        <?php if ($this->page > 1): ?>
-          <a href="<?=$urlBase . '&amp;facetpage=' . ($this->page - 1) . '&amp;facetsort=' . urlencode($this->sort) ?>" class="facet js-facet-prev-page" data-page="<?=($this->page + 1) ?>" data-sort="<?=$this->sort ?>" data-limit="<?=count($this->data) ?>" data-lightbox-ignore><?=$this->translate('prev_ellipsis') ?></a>
-        <?php endif; ?>
-        <?php foreach ($this->data as $item): ?>
-          <?=$this->render('Recommend/SideFacets/single-facet.phtml', [
-            'exclude' => $this->exclude,
-            'facet' => $item,
-            'group' => $this->facet,
-            'url' => $this->results->getUrlQuery(),
-            'urlBase' => $searchAction,
-          ]) ?>
-        <?php endforeach; ?>
-      <?php endif; ?>
-      <?php if ($this->anotherPage): ?>
-        <a href="<?=$urlBase . '&amp;facetpage=' . ($this->page + 1) . '&amp;facetsort=' . urlencode($key) ?>" class="facet js-facet-next-page" data-page="<?=($this->page + 1) ?>" data-sort="<?=$this->escapeHtmlAttr($key) ?>" data-lightbox-ignore>
-          <span class="text"><?=$this->translate('more_ellipsis') ?></span>
-        </a>
-      <?php endif; ?>
+    <div class="full-facet-list facet-group<?php if(!$active): ?> hidden<?php endif; ?>" id="facet-list-<?=$this->escapeHtmlAttr($key) ?>">
+      <?php
+        //The content of this list will later be updated via AJAX, see facet-list-content.phtml -->
+        $params = $this->delegateParams;
+        $params['urlBase'] = $urlBase;
+        $params['searchAction'] = $searchAction;
+        $params['inLightbox'] = $this->inLightbox;
+      ?>
+      <?=$this->render('search/facet-list-content.phtml', $params) ?>
     </div>
   <?php endforeach; ?>
 </div>
 <button class="btn btn-default lightbox-only" data-dismiss="modal"><?=$this->translate('close') ?></button>
-<?=$this->inlineScript(\Laminas\View\Helper\HeadScript::SCRIPT, '(typeof VuFind.lightbox_facets !== "undefined") && VuFind.lightbox_facets.setup();', 'SET'); ?>
+
+<?php if ($this->inLightbox || $this->ajaxLightbox): ?>
+  <?=$this->inlineScript(\Laminas\View\Helper\HeadScript::SCRIPT, 'VuFind.lightbox_facets.setup();', 'SET')?>
+<?php endif; ?>
+<?=$this->inlineScript(\Laminas\View\Helper\HeadScript::SCRIPT, 'VuFind.facetList.setup();', 'SET')?>
```

Comment:
Since lightbox_facets appears to have a dependency on facetList, would it be better to load facetList first? (It probably doesn't matter in current practice, but just wondering if it would be more future-proof to reverse the order).